### PR TITLE
Update build.gradle

### PR DESCRIPTION
**Pull request by Wattos**

### CODE REVIEW

File: beekeeper-code-analysis-plugin/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 dependencies {
-    implementation 'gradle.plugin.com.github.spotbugs:spotbugs-gradle-plugin:2.0.0'
+    implementation 'gradle.plugin.com.github.spotbugs:spotbugs-gradle-plugin:4.5.0'
```
Comment (Wattos) [inline beekeeper-code-analysis-plugin/build.gradle:2]:
```suggestion
    implementation 'gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.5.0'
```

File: beekeeper-code-analysis-plugin/build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 dependencies {
-    implementation 'gradle.plugin.com.github.spotbugs:spotbugs-gradle-plugin:2.0.0'
+    implementation 'gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.5.0'
```
Comment (Wattos) [inline beekeeper-code-analysis-plugin/build.gradle:2]: new version

File: beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java [hunk 3]
```diff
@@ -54,8 +49,8 @@ private void configureTasks(Project project) {
                 task.setEnabled(false);
             } else {
                 boolean isJenkins = isJenkins();
-                task.getReports().getXml().setEnabled(isJenkins);
-                task.getReports().getHtml().setEnabled(!isJenkins);
+                task.getReports().maybeCreate("html").setEnabled(isJenkins);
```
Comment (isacandrei) [inline beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java:52]: In the old version this was like this: 
```suggestion
                task.getReports().maybeCreate("html").setEnabled(!isJenkins);
```

File: beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java [hunk 4]
```diff
@@ -54,8 +49,8 @@ private void configureTasks(Project project) {
...
+                task.getReports().maybeCreate("xml").setEnabled(isJenkins);
```
Comment (Wattos) [inline beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java:53]:
```suggestion
                task.getReports().maybeCreate("xml").setEnabled(!isJenkins);
```

File: beekeeper-formatter-plugin/src/test/groovy/io/beekeeper/formatter/GradleVersionCompatibilitySpec.groovy [hunk 5]
```diff
@@ -24,9 +24,15 @@ class GradleVersionCompatibilitySpec extends Specification {
         '4.9',
         '4.10',
         "5.0",
-        '5.1'
+        "5.1",
+        '5.6'
```
Comment (isacandrei) [inline beekeeper-formatter-plugin/src/test/groovy/io/beekeeper/formatter/GradleVersionCompatibilitySpec.groovy:28]: Shall we remove all the old versions?

File: beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java (hunk 3 above)
Comment (Wattos) [inline beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java:52]:
```suggestion
                task.getReports().maybeCreate("xml").setEnabled(isJenkins);
```

File: beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java [hunk 6]
```diff
@@ -54,8 +49,8 @@ private void configureTasks(Project project) {
                 task.setEnabled(false);
             } else {
                 boolean isJenkins = isJenkins();
-                task.getReports().getXml().setEnabled(isJenkins);
-                task.getReports().getHtml().setEnabled(!isJenkins);
+                task.getReports().maybeCreate("html").setEnabled(isJenkins);
+                task.getReports().maybeCreate("xml").setEnabled(!isJenkins);
```
Comment (Wattos) [inline beekeeper-code-analysis-plugin/src/main/java/io/beekeeper/gradle/code/CodeAnalysisPlugin.java:53]:
```suggestion
                task.getReports().maybeCreate("html").setEnabled(!isJenkins);
```

**Review by isacandrei — APPROVED** [review]

**Pull request merged**